Packs/CortexDataLake/Integrations/CortexDataLake/README.md

```diff
@@ -776,7 +776,8 @@ Searches the Cortex panw.threat table, which is the threat logs table for PAN-OS
 | File.Type | String | The file type, as determined by libmagic (same as displayed in file entries). | 
 
 
-##### Command Example
+##### Command Examples
+```!cdl-query-threat-logs query="is_packet_capture = true AND severity = \"Critical\"" fields=pcap limit=10```
```

Comment:
lets remove backslashes

Reply:
Are you sure? It won't work without the backslashes..